### PR TITLE
Breadcrumbs and title clean up

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary
* Removes topic title from TopicsPage that was never removed from designs
* Moves Classes routes under the home page
* Removes classes top tab
* Adds breadcrumbs for classes pages back in
* Moves topics specific breadcrumbs functionality into TopicsPage.vue
* Cleans up unused code

## References
No extant issues describing this.

## Reviewer guidance
Title clean up:
| Before | After |
| --------- | ------- |
| Screenshot from 2021-12-09 13-24-27 | Screenshot from 2021-12-09 13-45-07 |

Classes breadcrumbs and route nested under home:
classy_breadcrumbs

Topics breadcrumbs are covered by breadcrumbs.spec.js.

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [x] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by jtamiace — APPROVED** [review]
Looks great 🎉 could you bump up the topic titles that are left from `h3` to `h2`?

**Comment by rtibbles** [comment]
The ones above the cards for their children?

**Comment by jtamiace** [comment]
This one - basically to be the same size as the one you removed
Screen Shot 2021-12-09 at 5 36 25 PM

**Comment by rtibbles** [comment]
Updated:

Screenshot from 2021-12-09 17-39-31

File: kolibri/plugins/learn/assets/src/composables/__mocks__/useSearch.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/**
+ * `useSearch` composable function mock.
```
Comment (nucleogenesis) [inline kolibri/plugins/learn/assets/src/composables/__mocks__/useSearch.js:2]: I really appreciate these comments. I feel so much more confident going into writing tests around this than any Vuex module and these comments are a great guide

**Review by nucleogenesis — APPROVED** [review]
Code looks great and really appreciate having these breadcrumbs here.

File: kolibri/plugins/learn/assets/src/composables/__mocks__/useSearch.js (hunk 1 above)
Comment (rtibbles) [reply kolibri/plugins/learn/assets/src/composables/__mocks__/useSearch.js:2]: This is copy pasted (and then edited for this specific context) from the template created by @MisRob, so she deserves all the credit!

**Pull request merged**